examples/next-lmnt/app/api/chat/route.ts

```diff
@@ -0,0 +1,60 @@
+import {
+  OpenAIStream,
+  StreamingTextResponse,
+  experimental_StreamData,
+  experimental_forwardLmntSpeechStream,
+} from 'ai';
+import Speech from 'lmnt-node';
+import OpenAI from 'openai';
+
+// Create an OpenAI API client
+const openai = new OpenAI({
+  apiKey: process.env.OPENAI_API_KEY || '',
+});
+
+// Create an LMNT API client
+const speech = new Speech(process.env.LMNT_API_KEY || 'no key');
+
+// Note: The LMNT SDK does not work on edge yet (as of v1.1.2)
+// export const runtime = 'edge';
```

Comment:
Hi @lgrammel FYI @kaikato just merged a short README in `lmnt-node` describing how we got this working -- if you see an even better way let us know, but with the one change to the `next.config.js` file it should work with `edge`. https://github.com/lmnt-com/lmnt-node/pull/32

Reply:
I thought to add that when we were hacking on https://github.com/vercel/ai-chatbot/pull/151 I did see issues that looked like a challenge re: websockets staying alive on `edge` and so for deployment I switched to `nodejs` and didn't look further at the time. You can see the deployment focused work I did atop that PR here: https://github.com/shaper/lmnt-ai-chatbot/commits/main/

Reply:
@lgrammel @MaxLeiter When TTS will come?